edkrepo/common/logger.py

```diff
@@ -0,0 +1,143 @@
+import os
+import re
+import git
+import sys
+import time
+import errno
+import string
+import logging
+
+from colorama import init
+from datetime import date
+
+from edkrepo.common.humble import LINE_BREAK, REMOVE_LOG_FAILED, DISK_SPACE_ERROR
+from edkrepo.common.pathfix import expanduser
+from edkrepo.common.edkrepo_exception import EdkrepoLogsRemoveException
+from edkrepo.config.config_factory import GlobalUserConfig
+
+init(autoreset=True)
+
+class CustomFormatter(logging.Formatter):
+    yellow = "\x1b[33;1m"

```

Comment:
Where do the color codes come from can we use built in colorama color constants to define this?

Reply:
Replaced color codes with colorama color variables